### PR TITLE
Add RangeMC.net

### PR DESCRIPTION
**Pull request by RootException**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/rangemc/background.png
Comment (maaaathis) [inline minecraft_servers/rangemc/background.png:1]: Compress this file

File: minecraft_servers/rangemc/background@2x.png
Comment (maaaathis) [inline minecraft_servers/rangemc/background@2x.png:1]: Compress this file

File: minecraft_servers/rangemc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "nice_name": "RangeMC",
+  "direct_ip": "rangemc.net",
+  "supported_languages": [
+     "en",
+     "de"
+  ],
+  "social": {
+    "web": "https://rangemc.net",
+    "web_shop": "https://rangemc.net/store",
+    "web_support": "https://rangemc.net/support",
```
Comment (maaaathis) [inline minecraft_servers/rangemc/manifest.json:12]: Cannot be hidden behind a login mask; provide a classic faq or contact form or remove

File: minecraft_servers/rangemc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "twitter": "RangeMCnet",
+    "tiktok": "RangeMC",
+    "instagram": "rangemc",
+    "discord": "https://discord.gg/8x4uSpp7UM",
+    "youtube": "https://www.youtube.com/@rangemc"
+  },
+  "gamemodes": {
+    "citybuild": {
+      "name": "JumpWorld",
```
Comment (maaaathis) [inline minecraft_servers/rangemc/manifest.json:21]: key doesn't match to name

File: minecraft_servers/rangemc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  "gamemodes": {
+    "citybuild": {
+      "name": "JumpWorld", 
+      "color": "#ffaa00"
+    },
+    "dna": {
+      "name": "DNA",
+      "color": "#ADD8E6",
+      "url": "https://www.timolia.de/games#dna",
```
Comment (maaaathis) [inline minecraft_servers/rangemc/manifest.json:27]: I don't think that's what you want here...

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by RootException** [comment]
Suggestions Done

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**